pages/flashbar/collapsible.visual-tests.page.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import Flashbar from '~components/flashbar';
+import React from 'react';
+import { Box } from '~components';
+import ScreenshotArea from '../utils/screenshot-area';
+import { FocusTarget, sampleNotifications } from './common';
+
+const items = [
+  { ...sampleNotifications.warning, id: '0' },
+  { ...sampleNotifications.error, id: '1' },
+  {
+    ...sampleNotifications.success,
+    id: '2',
+  },
+];
+
+export default function CollapsibleFlashbarVisualTests() {
+  return (
+    <>
+      <h1>Collapsible Flashbar visual tests</h1>
+      <FocusTarget />
```

Comment:
This element allows to expand the Flashbar with the keyboard so that the focus ring is also covered in the tests.